requirements-noaws.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.16.0
```

Comment:
Hi David, that would be my favorite. But can we configure things so that the default (used by pip install) is to install with AWS? So that installation without AWS is a special case for developers?